### DISCUSSION TITLE
[Duplicated] Replace Babel Compiler with SWC

### DISCUSSION THREAD

**Discussion by RobertLowe** [meteor/meteor#11738] [Feature Requests]
It's got great performance over babel

> It's 20x faster than babel on single thread, and 70x faster on 4 core benchmark
> https://swc.rs/

https://github.com/swc-project/swc

We should consider leveraging swc

**Comment by StorytellerCZ** [comment]
This is a duplicate of this discussion: https://github.com/meteor/meteor/discussions/11587

**Comment by nachocodoner** [comment]
We delivered SWC already Meteor 3.3 with faster builds/rebuilds x3. See [blog post](https://dev.to/meteor/faster-builds-in-meteor-33-modern-build-stack-with-swc-and-bundler-optimizations-fm2). ☄️🚀

Closing this discussion as it's a duplicate. In https://github.com/meteor/meteor/discussions/11587 we keep updates on the plan and deliveries on Meteor development experience improvements.
